Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM quay.io/jupyter/minimal-notebook:afe30f0c9ad8
```

Comment:
Since we will use this as the base image, we should be using Python 3.11 and not Python 3.12 in our `environment.yml` and `conda-lock` file to avoid issues with the container.

Reply:
I will try the Dockerfile tomorrow then. However, I am unsure about the issue with space when building the image since I did not have something like that happen. I will let you know if I have a similar issue since it could be indicative.

Reply:
thx - i think it's just my computer running outta space - not a bug or anything. sorry! :$ i'll comment here tomorrow once i'm able to push and change the issue statuses back to 'ready for review'.

Reply:
Alright, I will check the issues from time to time tomorrow then.